jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/helper/DockerFileUtil.java

```diff
@@ -312,4 +317,23 @@ private static String trimSurroundingQuotes(String argStringValue) {
         }
         return argStringValue;
     }
+
+    static List<String> getPortsFromDockerFile(File dockerFile) {
+        Properties properties = new Properties();
+        try {
+            return getPortsFromDockerFileLines(extractLines(dockerFile, "EXPOSE", properties, null));
+        } catch (IOException ioException) {
+            throw new IllegalArgumentException("Error in reading Dockerfile", ioException);
+        }
+    }
+
+    static List<String> getPortsFromDockerFileLines(List<String[]> dockerLinesContainingExpose) {
```

Comment:
If this is visible for testing, make it private and test using the dependent method (should be easy enough).

Also, nit: rename it to be consistent: `extractPorts`